lambdas/service/app.py

```diff
@@ -624,7 +628,7 @@ def validate_filters(filters):
         raise BRE('The `filters` parameter must be a dictionary')
     field_types = app.repository_controller.field_types(app.catalog)
     for field, filter_ in filters.items():
-        validate_field(field)
+        validate_field(field, 'accessible')
```

Comment:
```suggestion
        validate_field(field, *app.synthetic_fields)
```
This, or a modification to `validate_field` as suggested below.